### PR TITLE
add filter, modify author in setup.cfg

### PR DESCRIPTION
**Pull request by datelier**
SSIA

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,7 @@ SHADOW_ROOT = vald
 
 PROTOS = \
 	v1/agent/core/agent.proto \
+	v1/filter/ingress/ingress_filter.proto \
```
Comment (kpango) [inline Makefile:41]: How about adding an egress filter as well?

Comment (datelier) [reply Makefile:41]: Sounds good :+1:

**Review by kpango — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM!!

**Pull request merged**